t/venom/lib/check_host_internet_access.yml

```diff
@@ -0,0 +1,11 @@
+executor: check_host_internet_access
+input:
+  host: "{{.node01_mgmt_ip}}"
+  user: "{{.ssh_user}}"
+steps:
+  - type: ssh
+    host: '{{.input.host}}'
+    user: '{{.input.user}}'
+    command: |
+      cd /usr/local/pf/t/venom ; \
+      sudo /usr/local/pf/t/venom/venom-wrapper.sh test_suite//common/check_internet_access.yml
```

Comment:
I think you lost path to `nodes` directory here.

Reply:
Not exactly.
The main idea is to remove the `node` directory and just use `executor` and `test_suites/common`. But to be discuss. It really depends how we will use executors.
It could be a good move to remove extra specific layers.

Reply:
- Full paths

Reply:
Done add vars